server/proxy/install.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 // InstallApp installs an App.
-//  - cc is the Context that will be passed down to the App's OnInstall callback.
+//   - cc is the Context that will be passed down to the App's OnInstall callback.
```

Comment:
There are no changes here, but it's just the autoformat that triggered this change probably :-)

Reply:
Yes, that is due to the go 1.19 doc updates (https://tip.golang.org/doc/go1.19)